test/src/kbasesearchengine/test/integration/SearchAPIIntegrationTest.java

```diff
@@ -629,8 +627,52 @@ public void pruneNarrative() throws Exception {
         assertThat("incorrect keyprops", od.getKeyProps(),
                 is(ImmutableMap.of("title", "a title")));
     }
-    
-    /* ****** Auth client tests - to be moved to their own suite *** 
+
+
+    @Test
```

Comment:
This seems like it'd be a better fit for a unit test rather than an integration test.

Reply:
I prefer the integration test because the verification code relies on the ES response. If I mock the ES error response and we change ES version in the future to a version that has a slightly different message, the unit test will silently pass each time and the code may become buggy.

Reply:
Let me rephrase - this seems like it belongs in ElasticIndexingStorageTest (which aren't unit tests, strictly speaking) rather than the API integration test

Reply:
Moved to ElasticIndexingStorageTest.